limited/ravenkeeper

```diff
@@ -16,10 +16,10 @@ __Formalized__
 Starting: 
  • Set Counter to floor $total/5
  • Display `Ravens` (Counter)
-Immediate Night: [Condition: @Self->Counter > `0`] |ravenkeeper.1|
+Immediate Night: [Condition: (@Self->Counter > `0`) and (not (@Selection is @Self))] |ravenkeeper.1|
  • Apply `Raven` to @Selection (~Persistent)
  • Decrement Counter
-Immediate Day: [Condition: @Selection has `Raven`] |ravenkeeper.2|
+Immediate Day: [Condition: @Selection has `Raven`, Temporal: Day 2+] |ravenkeeper.2|
```

Comment:
Why is removing a raven not allowed on Day 1?

Reply:
Oops meant to be D1+

Reply:
Appears to be resolved by https://github.com/WerewolvesRevamped/Werewolves-Roles/pull/1542/commits/0c82373f6c0c5d850ea03c6f587ed2bc6ebaf85a